docs/ch12-qtquick3d/src/basicscene/main.qml

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2023, Johan Thelin
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the editors nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import QtQuick
+import QtQuick3D
+
+Window {
+    width: 640
+    height: 480
+    visible: true
+    title: qsTr("Basic Scene")
+
+    View3D {
+        anchors.fill: parent
+
+// #region environment
+        environment: SceneEnvironment {
+            clearColor: "#222222"
+            backgroundMode: SceneEnvironment.Color
+        }
+// #endregion environment
+
+// #region meshes
+        Model {
+            position: Qt.vector3d(0, 0, 0)
+            scale: Qt.vector3d(1, 1.25, 1)
+            source: "#Cone"
+            materials: [ DefaultMaterial { diffuseColor: "yellow"; } ]
```

Comment:
```suggestion
            materials: [ PrincipledMaterial { baseColor: "yellow"; } ]
```

Reply:
Is there a reason for suggesting this? Is it from a pure esthetics point of view, or for some technical reason? Right now, I stick with the DefaultMaterial until materials are discussed in more details?

Reply:
DefaultMaterial was a compatibility material with Qt 3D Studio with an unfortunate name that implies it the "Default" choice, which it shouldn't be.  Ideally it will be replaced with something else that actually lives up the the empty promises the documentation makes about what it is (SpecularGlossyMaterial or something).  PrincipledMaterial is the *real* "default" material ;-)